devnet/alice.Dockerfile

```diff
@@ -4,7 +4,8 @@
 FROM golang:1.17
 
 ARG DD_API_KEY=somekey
-RUN DD_API_KEY=${DD_API_KEY}  DD_AGENT_MAJOR_VERSION=7 DD_INSTALL_ONLY=true DD_SITE="datadoghq.com" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"
+ENV DD_API_KEY=${DD_API_KEY}
```

Comment:
Note this will persist in the image, so make sure the image is not publicly pullable